packages/hint-validate-set-cookie-header/src/hint.ts

```diff
@@ -296,20 +295,22 @@ export default class ValidateSetCookieHeaderHint implements IHint {
                     parsedSetCookie = parse(setCookieHeader);
                     parsedSetCookie.resource = resource;
                 } catch (err) {
-                    await context.report(resource, err.message, { element });
+                    context.report(resource, err.message, { element });
 
                     return;
                 }
 
-                for (const defaultValidator of defaultValidators) {
+                defaultValidators.every((defaultValidator) => {
```

Comment:
I think we should do a `forEach` in here, otherwise we will just report the first batch of `messages` and stop processing right after which is not what we were doing before.

Reply:
If that the right behavior? I mean, in the old code, we stopped after the first validation failed, that's why I'm using an `every`

Reply:
You are right. I missed the `return` right after the `await`.